packages/docs/blog/2023-08-08-system-strings.md

```diff
@@ -0,0 +1,206 @@
+---
+slug: demystifying-fhir-system-strings
+title: Demystifying FHIR System Strings
+authors:
+  name: Rahul Agarwal
+  title: Medplum Core Team
+  url: https://github.com/rahul1
+  image_url: https://github.com/rahul1.png
+  tags: [fhir-datastore]
+---
+
+import CodeBlock from '@theme/CodeBlock'
+
+# Demystifying FHIR System Strings
+
+One of the main sources of confusion when starting out with FHIR are `system` strings.
+
+This field is ubiquitous across FHIR elements, but many developers who are new to healthcare don't understand its purpose or how to set it properly.
+
+So today, we're going to delve into `system` strings to understand what they're for and how to use them!
+
+System strings are commonly found on two distinct element types:
+
+- [`Identifiers`](/docs/fhir-basics#identifiers-naming-resources)
+- [`CodeableConcepts`](/docs/fhir-basics#codeable-concepts-standarding-data)
+
```

Comment:
If you have loaded up your sample data, then you can see some examples in the [Patient](app.medplum.com/Patient) resource page for identifiers and [MedicationRequest](app.medplum.com/MedicationRequest) resource page for Codable concepts.